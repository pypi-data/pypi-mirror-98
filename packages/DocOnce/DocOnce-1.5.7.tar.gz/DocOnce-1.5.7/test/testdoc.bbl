\begin{thebibliography}{10}

\bibitem{Glimsdal_et_al_20006}
S.~Glimsdal, G.~Pedersen, K.~Atakan, C.~B. Harbitz, H.~P. Langtangen, and
  F.~L{\o}vholt.
\newblock Propagation of the {Dec.~26}, 2004 {Indian Ocean Tsunami}: Effects of
  dispersion and source characteristics.
\newblock {\em International Journal of Fluid Mechanics Research},
  33(1):15--43, 2006.

\bibitem{Haga_et_al_2011a}
J.~B. Haga, H.~Osnes, and H.~P. Langtangen.
\newblock On the causes of pressure oscillations in low-permeable and
  low-compressible porous media.
\newblock {\em International Journal of Analytical and Numerical Methods in
  Geomechanics}, 2011.

\bibitem{Jeberg_et_al_2004}
P.~V. Jeberg, H.~P. Langtangen, and C.~B. Terp.
\newblock Optimization with {Diffpack}: Practical example from welding.
\newblock Report 2004-12, Simula Research Laboratory, 2004.

\bibitem{Langtangen:85}
H.~P. Langtangen.
\newblock Solution of the {Navier-Stokes} equations with the finite element
  method in two and three dimensions, 1985.
\newblock Cand.Scient. thesis.

\bibitem{Langtangen_1989e}
H.~P. Langtangen.
\newblock Computational methods for two-phase flow in oil reservoirs, 1989.
\newblock Dr.~Scient.~thesis.

\bibitem{Langtangen_1988d}
H.~P. Langtangen.
\newblock The {FEMDEQS} program system.
\newblock Research report in mechanics, Department of Mathematics, University
  of Oslo, 1989.

\bibitem{Langtangen_1992c}
H.~P. Langtangen.
\newblock Stochastic breakthrough time analysis of an enhanced oil recovery
  process.
\newblock {\em SIAM Journal on Scientific Computing}, 13:1394--1417, 1992.

\bibitem{Langtangen:94b}
H.~P. Langtangen.
\newblock {Diffpack}: Software for partial differential equations.
\newblock In A.~Vermeulen, editor, {\em Proceedings of the Second Annual
  Object-Oriented Numerics Conference (OON-SKI'94), Sunriver, Oregon, USA},
  1994.

\bibitem{Langtangen_1994a}
H.~P. Langtangen.
\newblock Numerical solution of first passage problems in random vibrations.
\newblock {\em SIAM Journal of Scientific and Statistical Computing},
  15:997--996, 1994.

\bibitem{Langtangen_2003a}
H.~P. Langtangen.
\newblock {\em Computational Partial Differential Equations - Numerical Methods
  and {Diffpack} Programming}.
\newblock Texts in Computational Science and Engineering. Springer, second
  edition, 2003.

\bibitem{Langtangen_talk_2007a}
H.~P. Langtangen.
\newblock Computational modeling of huge tsunamis from asteroid impacts.
\newblock Invited keynote lecture at the \emph{International conference on
  Computational Science 2007 (ICCS'07)}, Beijing, China, May 2007.

\bibitem{Langtangen_2008a}
H.~P. Langtangen.
\newblock {\em {P}ython Scripting for Computational Science}.
\newblock Texts in Computational Science and Engineering. Springer, third
  edition, 2008.

\bibitem{Langtangen_2012}
H.~P. Langtangen.
\newblock {\em A Primer on Scientific Programming With {P}ython}.
\newblock Texts in Computational Science and Engineering. Springer, third
  edition, 2012.

\bibitem{Langtangen_et_al_2002}
H.~P. Langtangen, K.-A. Mardal, and R.~Winther.
\newblock Numerical methods for incompressible viscous flow.
\newblock {\em Advances in Water Resources}, 25:1125--1146, 2002.

\bibitem{Langtangen:95}
H.~P. Langtangen and G.~Pedersen.
\newblock {\em Finite Elements for the {Boussinesq} Wave Equations}, pages
  117--126.
\newblock Kluwer Academic Publishers, 1995.

\bibitem{Langtangen_Pedersen_2002}
H.~P. Langtangen and G.~Pedersen.
\newblock Propagation of large destructive waves.
\newblock {\em International Journal of Applied Mechanics and Engineering},
  7(1):187--204, 2002.

\bibitem{Langtangen:91}
H.~P. Langtangen and A.~Tveito.
\newblock Numerical methods in continuum mechanics.
\newblock Si-report, Center for Industrial Research, 1991.
\newblock Lecture notes for a course (ME-IN 324). 286 pages.

\bibitem{Mardal_et_al_2003a}
K.-A. Mardal, G.~W. Zumbusch, and H.~P. Langtangen.
\newblock Software tools for multigrid methods.
\newblock In H.~P. Langtangen and A.~Tveito, editors, {\em Advanced Topics in
  Computational Partial Differential Equations -- Numerical Methods and
  Diffpack Programming}, Lecture Notes in Computational Science and
  Engineering, pages 97--152. Springer, 2003.

\bibitem{Mortensen_et_al_2011}
M.~Mortensen, H.~P. Langtangen, and G.~N. Wells.
\newblock A {FEniCS}-based programming framework for modeling turbulent flow by
  the {Reynolds}-averaged {Navier-Stokes} equations.
\newblock {\em Advances in Water Resources}, 34(9), 2011.

\bibitem{Rahman_et_al_2006b}
S.~Rahman, J.~Gorman, C.~H.~W. Barnes, D.~A. Williams, and H.~P. Langtangen.
\newblock Numerical investigation of a piezoelectric surface acoustic wave
  interaction with a one-dimensional channel.
\newblock {\em Physical Review B: Condensed Matter and Materials Physics}, 74,
  2006.
\newblock 035308.

\end{thebibliography}
