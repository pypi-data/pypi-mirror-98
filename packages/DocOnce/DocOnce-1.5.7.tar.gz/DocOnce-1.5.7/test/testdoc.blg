This is BibTeX, Version 0.99d (TeX Live 2017/Debian)
Capacity: max_strings=100000, hash_size=100000, hash_prime=85009
The top-level auxiliary file: testdoc.aux
The style file: plain.bst
Database file #1: papers.bib
Warning--entry type for "Langtangen_1989e" isn't style-file defined
--line 170 of file papers.bib
Warning--entry type for "Langtangen:85" isn't style-file defined
--line 179 of file papers.bib
Warning--can't use both author and editor fields in Langtangen:95
You've used 20 entries,
            2118 wiz_defined-function locations,
            628 strings with 7872 characters,
and the built_in function-call counts, 5860 in all, are:
= -- 570
> -- 272
< -- 2
+ -- 110
- -- 87
* -- 357
:= -- 1024
add.period$ -- 66
call.type$ -- 20
change.case$ -- 104
chr.to.int$ -- 0
cite$ -- 21
duplicate$ -- 223
empty$ -- 471
format.name$ -- 87
if$ -- 1231
int.to.chr$ -- 0
int.to.str$ -- 20
missing$ -- 18
newline$ -- 105
num.names$ -- 44
pop$ -- 108
preamble$ -- 1
purify$ -- 82
quote$ -- 0
skip$ -- 157
stack$ -- 0
substring$ -- 269
swap$ -- 36
text.length$ -- 2
text.prefix$ -- 0
top$ -- 0
type$ -- 72
warning$ -- 1
while$ -- 52
width$ -- 22
write$ -- 226
(There were 3 warnings)
