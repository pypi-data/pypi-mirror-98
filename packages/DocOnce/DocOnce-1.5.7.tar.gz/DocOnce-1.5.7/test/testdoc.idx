\indexentry{somefunc@{\rm\texttt{somefunc}} function|hyperpage}{4}
\indexentry{figures|hyperpage}{9}
\indexentry{movies|hyperpage}{11}
\indexentry{test index with verbatim text@test index with {\rm\texttt{verbatim text}} which is possible|hyperpage}{14}
\indexentry{test two@test {\rm\texttt{two}} (separate) {\rm\texttt{verbatim expressions}} which is also possible|hyperpage}{14}
\indexentry{index with!subindex|hyperpage}{14}
\indexentry{\textbf{boldface word} in index|hyperpage}{14}
\indexentry{index with \textbf{boldface word}|hyperpage}{14}
\indexentry{index with!\textbf{boldface word} in subentry|hyperpage}{14}
\indexentry{double \textbf{boldface word}! \textbf{boldface word} in subentry too|hyperpage}{14}
\indexentry{index, with comma, and one more|hyperpage}{14}
