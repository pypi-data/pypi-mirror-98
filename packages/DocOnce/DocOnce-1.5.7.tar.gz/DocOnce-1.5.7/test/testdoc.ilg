This is makeindex, version 2.15 [TeX Live 2017] (kpathsea + Thai support).
Scanning input file testdoc.idx....done (11 entries accepted, 0 rejected).
Sorting entries....done (41 comparisons).
Generating output file testdoc.ind....done (37 lines written, 0 warnings).
Output written in testdoc.ind.
Transcript written in testdoc.ilg.
