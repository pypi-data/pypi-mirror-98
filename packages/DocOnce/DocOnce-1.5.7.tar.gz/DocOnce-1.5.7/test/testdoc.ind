\begin{theindex}

  \item \textbf{boldface word} in index, \hyperpage{15}

  \indexspace

  \item double \textbf{boldface word}
    \subitem  \textbf{boldface word} in subentry too, \hyperpage{15}

  \indexspace

  \item figures, \hyperpage{11}

  \indexspace

  \item index with
    \subitem \textbf{boldface word} in subentry, \hyperpage{15}
    \subitem subindex, \hyperpage{15}
  \item index with \textbf{boldface word}, \hyperpage{15}
  \item index, with comma, and one more, \hyperpage{15}

  \indexspace

  \item movies, \hyperpage{12}

  \indexspace

  \item {\rm\texttt{somefunc}} function, \hyperpage{3}

  \indexspace

  \item test index with {\rm\texttt{verbatim text}} which is possible, 
		\hyperpage{15}
  \item test {\rm\texttt{two}} (separate) {\rm\texttt{verbatim expressions}} which is also possible, 
		\hyperpage{15}

\end{theindex}
