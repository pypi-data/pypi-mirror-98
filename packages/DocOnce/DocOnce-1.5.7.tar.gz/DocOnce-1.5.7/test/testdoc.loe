\contentsline {doconceexercise}{Example 1: Examples can be typeset as exercises}{18}{doconceexercisecounter.1}
\contentsline {doconceexercise}{Problem 2: Flip a Coin}{22}{doconceexercisecounter.2}
\contentsline {doconceexercise}{Exercise 3: Test of plain text exercise}{23}{doconceexercisecounter.3}
\contentsline {doconceexercise}{Project 4: Compute a Probability}{23}{doconceexercisecounter.4}
\contentsline {doconceexercise}{Project 5: Explore Distributions of Random Circles}{24}{doconceexercisecounter.5}
\contentsline {doconceexercise}{Exercise 6: Determine some Distance}{24}{doconceexercisecounter.6}
\contentsline {doconceexercise}{Some exercise without the "Exercise:" prefix}{26}{Item.10}
\contentsline {doconceexercise}{Exercise 7: Solution of differential equation}{26}{doconceexercisecounter.7}
\contentsline {doconceexercise}{Example 8: Just an example}{27}{doconceexercisecounter.8}
\contentsline {doconceexercise}{Exercise 9: Make references to projects and problems}{27}{doconceexercisecounter.9}
\contentsline {doconceexercise}{Project 10: References to Project\nobreakspace {}\vref {demo:ex:2} in a heading works for pdflatex}{27}{doconceexercisecounter.10}
