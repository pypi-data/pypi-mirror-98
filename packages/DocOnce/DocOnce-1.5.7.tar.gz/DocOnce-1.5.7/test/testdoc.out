\BOOKMARK [1][-]{section.1}{Section 1}{}% 1
\BOOKMARK [2][-]{subsection.1.1}{Subsection 1}{section.1}% 2
\BOOKMARK [2][-]{subsection.1.2}{Subsection 2: Testing figures}{section.1}% 3
\BOOKMARK [2][-]{subsection.1.3}{The \040parameter \(not ?\)}{section.1}% 4
\BOOKMARK [2][-]{subsection.1.4}{Custom Environments}{section.1}% 5
\BOOKMARK [2][-]{subsection.1.5}{Tables}{section.1}% 6
\BOOKMARK [2][-]{subsection.1.6}{A test of verbatim words in heading with subscript ai: !my\137file\137v1! and !my\137file\137v2! }{section.1}% 7
\BOOKMARK [2][-]{subsection.1.7}{Just bold}{section.1}% 8
\BOOKMARK [2][-]{subsection.1.8}{Just emphasize}{section.1}% 9
\BOOKMARK [2][-]{subsection.1.9}{Just verbatim }{section.1}% 10
\BOOKMARK [2][-]{subsection.1.10}{Bold beginning}{section.1}% 11
\BOOKMARK [2][-]{subsection.1.11}{Emphasize beginning}{section.1}% 12
\BOOKMARK [2][-]{subsection.1.12}{Verbatim beginning}{section.1}% 13
\BOOKMARK [2][-]{subsection.1.13}{Maybe bold end}{section.1}% 14
\BOOKMARK [2][-]{subsection.1.14}{Maybe emphasize end}{section.1}% 15
\BOOKMARK [2][-]{subsection.1.15}{Maybe verbatim end }{section.1}% 16
\BOOKMARK [2][-]{subsection.1.16}{The middle has bold word}{section.1}% 17
\BOOKMARK [2][-]{subsection.1.17}{The middle has emphasize word}{section.1}% 18
\BOOKMARK [2][-]{subsection.1.18}{The middle has verbatim word}{section.1}% 19
\BOOKMARK [2][-]{subsection.1.19}{Bibliography test}{section.1}% 20
\BOOKMARK [2][-]{subsection.1.20}{User-defined environments}{section.1}% 21
\BOOKMARK [2][-]{subsection.1.21}{URLs}{section.1}% 22
\BOOKMARK [2][-]{subsection.1.22}{Test of Some LaTeX Fixes}{section.1}% 23
\BOOKMARK [1][-]{section.2}{LaTeX Mathematics}{}% 24
\BOOKMARK [1][-]{section.3}{Exercises}{}% 25
\BOOKMARK [2][-]{subsection.3.1}{Not an exercise}{section.3}% 26
\BOOKMARK [1][-]{section.4}{Here goes another section}{}% 27
\BOOKMARK [1][-]{section.5}{More Exercises}{}% 28
\BOOKMARK [1][-]{section*.72}{References}{}% 29
\BOOKMARK [1][-]{appendix.A}{Just for testing; part I}{}% 30
\BOOKMARK [2][-]{subsection.A.1}{A subsection within an appendix}{appendix.A}% 31
\BOOKMARK [1][-]{appendix.B}{Just for testing; part II}{}% 32
\BOOKMARK [2][-]{subsection.B.1}{Appendix: Testing identical titles}{appendix.B}% 33
\BOOKMARK [2][-]{subsection.B.2}{Appendix: Testing identical titles}{appendix.B}% 34
\BOOKMARK [2][-]{subsection.B.3}{Appendix: Testing identical titles}{appendix.B}% 35
\BOOKMARK [2][-]{subsection.B.4}{Appendix: Testing identical titles}{appendix.B}% 36
\BOOKMARK [2][-]{subsection.B.5}{Appendix: Testing inline comments}{appendix.B}% 37
\BOOKMARK [2][-]{subsection.B.6}{Appendix: Testing headings ending with verbatim inline }{appendix.B}% 38
\BOOKMARK [1][-]{section*.76}{Index}{}% 39
